lib/utilities.js

```diff
@@ -12,8 +12,8 @@ function camelCase(string) {
     throw new TypeError('First argument must be a string');
   }
 
-  // no hyphen found
-  if (string.indexOf('-') === -1) {
+  // customStyle or no hyphen found
+  if (string.startsWith('--') || string.indexOf('-') === -1) {
```

Comment:
**fix**: do you mind testing the match using regex since [startsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith#Browser_compatibility) is not supported in IE?

```js
// regex that matches custom property or no hyphen
var CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX = /^--[a-zA-Z]+$|^[^-]+$/;

function camelCase(string) {
  // ...

  if (CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX.test(string)) {
    return string;
  }

  // ...
```

Reply:
seems to **not** work with custom styles having lisp-case/dash-case in between ... 

for example:

```
CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX.test('customStyle') // returns : true
CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX.test('--customStyle') // returns : true
CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX.test('--custom-style') // returns : false
```
I think it should also be possible to use custom style having dashes in between (e.g. '--custom-style'). Therefore lets consider an easier approach:

```
// customStyle or no hyphen found
  if (string.indexOf('--') === 0 || string.indexOf('-') === -1) {
    return string;
  }
```

It could be possible to use a regex for this check, or to directly replace the string by camelCase if needed, but since javascript does not fully support lookbehind-assertions, the regex gets a little tricky ...

Reply:
Good point about the lisp-case. What about the modified regex?

```diff
-var CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX = /^--[a-zA-Z]+$|^[^-]+$/;
+var CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX = /^--[a-zA-Z-]+$|^[^-]+$/;
```

The `indexOf` also works, but do you think it's worth making the declaration value check strict?